kubernetes/kubernetes.go

```diff
@@ -470,13 +475,18 @@ func NewNotFound(name, group, resource string) error {
 }
 
 // GetSelfSubjectAccessReview provides information on Kiali permissions
-func (in *K8SClient) GetSelfSubjectAccessReview(namespace, api, resourceType string, verbs []string) ([]*auth_v1.SelfSubjectAccessReview, error) {
+func (in *K8SClient) GetSelfSubjectAccessReview(ctx context.Context, namespace, api, resourceType string, verbs []string) ([]*auth_v1.SelfSubjectAccessReview, error) {
+	if config.Get().Server.Observability.Tracing.Enabled {
+		var span trace.Span
+		ctx, span = otel.Tracer(observability.TracerName).Start(ctx, "GetSelfSubjectAccessReview")
```

Comment:
I'd add custom tags in the Span with the method parameters:
![image](https://user-images.githubusercontent.com/1662329/144270321-56f3260d-f2fb-4202-bdbf-7e17eef8c737.png)

In the travels demo those are added in raw, but I guess there should be an API in the span interface.
That would help to identify the offending namespace/resource/object/etc.

Reply:
Yes you can add arbitrary attributes to the spans. I wasn't sure what all would be useful so left them out. Those can easily be added to the traces though. Adding the parameters seems like a good place to start.